CMakeLists.txt

```diff
@@ -71,4 +71,9 @@ else()
   option(BUILD_TESTING "Build the testing tree" ${BUILD_TESTING})
 endif()
 
+SET(METAIO_INSTALL_BIN_DIR ${ITK_INSTALL_BIN_DIR_CM24})
+SET(METAIO_INSTALL_LIB_DIR ${ITK_INSTALL_LIB_DIR_CM24})
+SET(METAIO_INSTALL_ARCHIVE_DIR ${METAIO_INSTALL_LIB_DIR})
+SET(METAIO_INSTALL_INCLUDE_DIR ${ITK_INSTALL_INCLUDE_DIR_CM24})
```

Comment:
The first two should be spelled `METAIO_INSTALL_RUNTIME_DIR` and `METAIO_INSTALL_LIBRARY_DIR`.

Reply:
Oh, yes, good catch. Strange that previous patch did affect the location of the library, though, when I tested it.